BrowserKit/Sources/Common/Extensions/URLExtension.swift

```diff
@@ -71,7 +71,9 @@ extension URL {
         if let range = host.range(of: "^(www|mobile|m)\\.", options: .regularExpression) {
             host.replaceSubrange(range, with: "")
         }
-        guard host != publicSuffix else { return nil }
+        // If the host equals the public suffix, it means that the host is already normalized.
```

Comment:
NIce one 👍 and well spotted 💪 
Not a blocker per se, but since I saw the `// Ecosia` checklist item checked I wonder whether this comment (alongside the commented line for the old implementation) should be part of it.

Reply:
NVM - Since it's Firefox does. I'm happy 😅

Reply:
Like I mentioned in the description, didn't add it since just copied from the latest Firefox 🙌 

>  Since this is already Firefox code, not adding any Ecosia comment as well.

Reply:
Yeah, it was me going thru the codebase directly 😅 . Thanks @lucaschifino